superset/config.py

```diff
@@ -614,18 +615,8 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
     CELERY_ACKS_LATE = False
     CELERY_ANNOTATIONS = {
         "sql_lab.get_sql_results": {"rate_limit": "100/s"},
```

Comment:
I think these removals break SEMVER - we shouldn't change the defaults for the configuration between majors. Let's keep the schedules, but perhaps add a deprecation?